owl/EASE-OBJ.owl

```diff
@@ -1876,6 +1885,16 @@
     
 
 
+    <!-- http://www.ease-crc.org/ont/EASE-OBJ.owl#FunctionalDesign -->
+
+    <owl:Class rdf:about="http://www.ease-crc.org/ont/EASE-OBJ.owl#FunctionalDesign">
+        <rdfs:subClassOf rdf:resource="http://www.ontologydesignpatterns.org/ont/dul/DUL.owl#Design"/>
+        <rdfs:comment>The design of an object from functionality point of view.</rdfs:comment>
```

Comment:
The comment is not very helpful as it does not really provide additional links to other things in the model. It is close to writing "A functional design is a functional design"